src/SFML/Window/Android/InputImpl.cpp

```diff
@@ -71,10 +71,160 @@ Keyboard::Scancode delocalize(Keyboard::Key /* key */)
+        case sf::Keyboard::Scan::I:                 return "I";
+        case sf::Keyboard::Scan::J:                 return "J";
+        case sf::Keyboard::Scan::K:                 return "K";
+        case sf::Keyboard::Scan::L:                 return "L";
+        case sf::Keyboard::Scan::M:                 return "M";
+        case sf::Keyboard::Scan::N:                 return "N";
+        case sf::Keyboard::Scan::O:                 return "O";
+        case sf::Keyboard::Scan::P:                 return "P";
+        case sf::Keyboard::Scan::Q:                 return "Q";
+        case sf::Keyboard::Scan::R:                 return "R";
+        case sf::Keyboard::Scan::S:                 return "S";
+        case sf::Keyboard::Scan::T:                 return "T";
+        case sf::Keyboard::Scan::U:                 return "U";
+        case sf::Keyboard::Scan::V:                 return "V";
+        case sf::Keyboard::Scan::W:                 return "W";
+        case sf::Keyboard::Scan::X:                 return "X";
+        case sf::Keyboard::Scan::Y:                 return "Y";
+        case sf::Keyboard::Scan::Z:                 return "Z";
+        case sf::Keyboard::Scan::Num1:              return "1";
+        case sf::Keyboard::Scan::Num2:              return "2";
+        case sf::Keyboard::Scan::Num3:              return "3";
+        case sf::Keyboard::Scan::Num4:              return "4";
+        case sf::Keyboard::Scan::Num5:              return "5";
+        case sf::Keyboard::Scan::Num6:              return "6";
+        case sf::Keyboard::Scan::Num7:              return "7";
+        case sf::Keyboard::Scan::Num8:              return "8";
+        case sf::Keyboard::Scan::Num9:              return "9";
+        case sf::Keyboard::Scan::Num0:              return "0";
+        case sf::Keyboard::Scan::Enter:             return "Enter";
+        case sf::Keyboard::Scan::Escape:            return "Escape";
+        case sf::Keyboard::Scan::Backspace:         return "Backspace";
+        case sf::Keyboard::Scan::Tab:               return "Tab";
+        case sf::Keyboard::Scan::Space:             return "Space";
+        case sf::Keyboard::Scan::Hyphen:            return "Hyphen (-)";
+        case sf::Keyboard::Scan::Equal:             return "Equal (=)";
+        case sf::Keyboard::Scan::LBracket:          return "Left Bracket [";
+        case sf::Keyboard::Scan::RBracket:          return "Right Bracket ]";
+        case sf::Keyboard::Scan::Backslash:         return "Backslash \\";
+        case sf::Keyboard::Scan::Semicolon:         return "Semicolon ;";
+        case sf::Keyboard::Scan::Apostrophe:        return "Apostrophe '";
+        case sf::Keyboard::Scan::Grave:             return "Grave `";
+        case sf::Keyboard::Scan::Comma:             return "Comma ,";
+        case sf::Keyboard::Scan::Period:            return "Period .";
+        case sf::Keyboard::Scan::Slash:             return "Slash /";
```

Comment:
Not sure it makes sense to have both the symbol and text.
For example when using the description the game settings for key mapping, I probably expect to just see the symbol.